### ISSUE TITLE
Feature request: provide Random seed of Random instance

### ISSUE THREAD

**Issue by michaldivis** [cryptic-wizard/random-word-generator#26]
Hi, I'd love to be able to provide the seed value for the `Random` used in this library or even the `Random` instance directly. I'm happy to implement it myself.

It could be used like this:

#### Provide seed
```C#
WordGenerator.UseSeed(123456); //configure seed prior to using the generator
WordGenerator myWordGenerator = new WordGenerator();
string word = myWordGenerator.GetWord(PartOfSpeech.noun);
```

#### Provide `Random` instance
```C#
WordGenerator.UseRandom(new Random()); //configure Random instance prior to using the generator
WordGenerator myWordGenerator = new WordGenerator();
string word = myWordGenerator.GetWord(PartOfSpeech.noun);
```

**Comment by michaldivis** [comment]
I've implemented this feature. Please consider merging the PR. Thanks!

**Comment by michaldivis** [comment]
Implemented and merged in #27
